src/components/birth-date-picker/birth-date-picker.tsx

```diff
@@ -0,0 +1,115 @@
+// TODO: use MobX.
+// TODO: filter input?
+// TODO: should we update the state when we receive a different value?
+// TODO: date validation belongs in a separate library.
+
```

Comment:
Generally, we strive not to have TODO comments. We either open new issue, or, if the matter is quick, just DO the TODO before PR. :-)